css/styles.css

```diff
@@ -55,39 +60,307 @@ a {
 	clear: both;
 }
 
+/* Photodiary */
+
+.photo-diary {
+	width: 85%;
+	padding-bottom: 100px;
+	margin: 0 auto;
+	color: rgba(115, 115, 115, .9);
+}
+
+.photo-diary span {
+	display: block;
+	font-size: 0.9rem;
+	font-family: 'Ubuntu', sans-serif;
+	text-transform: uppercase;
+	opacity: .6;
+}
+
+.photo-diary h1 {
+	padding-bottom: 10px;
+	font-family: "Playfair Display", serif;
+	font-weight: normal;
+}
+
+.photo-diary p {
+	display: block;
+	padding-bottom: 40px;
+	font-family: "Playfair Display", serif;
+	font-size: 0.9rem;
+}
+
+
+/* Div with 4 articles */
+
+.articles-div {
+	width: 85%;
+	padding-top: 80px;
+	padding-bottom: 40px;
+	margin: 0 auto;
+	color: rgba(115, 115, 115, .9);
+}
+
+.article img {
+	width: 90%;
+	padding-bottom: 30px;
+}
+
+.article span {
+	display: block;
+	font-family: 'Ubuntu', sans-serif;
+	font-size: 0.9rem;
+	text-transform: uppercase;
+	opacity: .6;
+}
+
+.article h1 {
+	font-family: "Playfair Display", serif;
+	font-weight: normal;	
+}
+
+.article p {
+	display: block;
+	padding-bottom: 30px;
+	font-family: "Playfair Display", serif;
+	font-size: 0.9rem;
+}
+
+
+/* Newsletter sign-up */
+
+.newsletter-div {
+	position: relative;
+	height: 260px;
+	background: #f2f2f2;
+	font-family: "Playfair Display", serif;
+	text-align: center;
+}
+
+.newsletter-div legend {
+	padding-top: 50px;
+	margin: 0 auto;
+	font-size: 1.4rem;
+}
+
+.newsletter-div input {
+	position: absolute;
+	top: 50%;
+	right: 20%;
+	display: inline-block;
+	width: 60%;
+	height: 30px;
+	border: none;
+	border-bottom: 2px solid #000;
+	background: #f2f2f2 url("../images/Send_icon.png") no-repeat right / 8%;
+	color: #000;
+	font-size: 0.8rem;
+}
+
+.newsletter-div input:focus {
+	outline: none;
+}
+
+
+.newsletter-div input::-webkit-input-placeholder {
+	color: #000;	
+}
+
+
+/* Button load-more */
+
+.btn-div {
+	padding: 80px 0;
+}
+
+.btn {
+	display: block;
+	width: 180px;
+	height: 60px;
+	padding: 10px 0;
+	border: 1px solid ;
+	margin: 0 auto;
+	font: 1.4rem "Playfair Display", serfif;
+	text-align: center;
+}
+
+ .btn-div a {
+ 		display: block;
+ }
+

```

Comment:
Najlepiej IMO wyrzucić całkiem wysokość. I dać równe paddingi z góry i z dołu. Dzięki temu tekst będzie na środku. Zmieniłbym też zdecydowanie ten button z `display: block;` na `inline-block`, w razie gdybyś chciał mieć 2 buttony koło siebie i wtedy nie potrzebujesz `margin: 0 auto;`. By wycentrować button ustaw `text-align: center;` dla `.btn-div`. Przy okazji wywal ustawioną na sztywno szerokość bo teraz jak dodasz trochę tekstu do buttona np. "Load more & more" to Ci się sypnie layout. Odstępy boczne też załatw paddingiem.

Reasumują - wywal `margin`, `width` i `height`, zmień na `display: inline-block` i `padding: .75em 1.5em` oraz daj `text-align: center;` dla `.btn-div`. Powinno śmigać tak jak chesz :)

P.S. Nie przejmuj się tym, że div jest o 1.5px wyższy niż w PSD-ku. Strona ma wyglądać wszędzie dobrze, nie identycznie co do piksela...